gdaladdo: add page (#8534)
tldr-pages/tldr: pages/common/gdaladdo.md

# gdaladdo

> Build overview images of raster datasets.
> More information: <https://gdal.org/programs/gdaladdo>.

> - Build overview images of a raster dataset using "average" resampling method:

Reviewer: ```suggestion
- Build overview images of a raster dataset using "average" [r]esampling method:
```

> - Build overview images of a raster dataset using the "average" [r]esampling method:

Reviewer 2: Should it be in quotes?
```suggestion
- Build overview images of a raster dataset using the "average" [r]esampling method:
```

> `gdaladdo -r {{average}} {{path/to/input.tif}}`

Reviewer 2: If the example description is for `average` the token syntax shouldn't be applied in the example, because it's not a placeholder.
```suggestion
`gdaladdo -r average {{path/to/input.tif}}`
```
